src/pspm_convert_pupil_gaze_distance.m

```diff
@@ -0,0 +1,159 @@
+function [sts, out] = pspm_convert_pupil_gaze_distance(fn, target, from, width, height, distance, options)
```

Comment:
Isn't it possible to combine somehow with `pspm_compute_visual_angle` and `pspm_convert_visangle2sps` ? Because it seems to have approximately the same pipeline. I feel like it's a bit redundant.

Reply:
I think pspm_compute_visual_angle is still valueable for getting just the degree angle data, thoughts @dominikbach ?